test/globals.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+global.chai = require('chai')
+global.should = global.chai.should()
+global.sinon = require('sinon')
+global.proxyquire = require('proxyquire')
```

Comment:
Have removed sinon and proxyquire references entirely, as we're not using them yet https://github.com/UKHomeOffice/hof-transpiler/pull/23/commits/9c77d76409b280d2e08972ac9174a78634a032ad